example/pages/_app.tsx

```diff
@@ -0,0 +1,184 @@
+import "../styles/globals.css"
+
+import { SigningCosmWasmClient } from "@cosmjs/cosmwasm-stargate"
+import { GasPrice } from "@cosmjs/stargate"
+import { Bech32Address } from "@keplr-wallet/cosmos"
+import { getKeplrFromWindow } from "@keplr-wallet/stores"
+import { ChainInfo } from "@keplr-wallet/types"
+import {
+  KeplrWalletConnectV1,
+  Wallet,
+  WalletManagerProvider,
+} from "@noahsaso/cosmodal"
+import type { AppProps } from "next/app"
+import { FunctionComponent } from "react"
+
+const LOCAL_STORAGE_KEY = "connectedWalletId"
+
+const MyApp: FunctionComponent<AppProps> = ({ Component, pageProps }) => (
+  <WalletManagerProvider
+    walletConnectClientMeta={{
+      name: "CosmodalExampleDApp",
+      description: "A dApp using the @noahsaso/cosmodal library.",
+      url: "https://noahsaso-cosmodal.vercel.app",
+      icons: ["https://moonphase.is/image.svg"],
+    }}
+    wallets={AvailableWallets}
+    renderLoader={() => <p>Loading...</p>}
+    localStorageKey={LOCAL_STORAGE_KEY}
+    saveToLocalStorageOnConnect
+    clearLocalStorageOnDisconnect
+    useLocalStorageForAutoConnect
+    // Auto connect if value is present. Check existence of localStorage
+    // to prevent server rendering issues.
+    attemptAutoConnect={
+      typeof localStorage !== "undefined" &&
+      !!localStorage.getItem(LOCAL_STORAGE_KEY)
+    }
+  >
+    <Component {...pageProps} />
+  </WalletManagerProvider>
+)
+
+export default MyApp
+
+const CHAIN_ID: string = "uni-3"
+const GAS_PRICE: string = "0.0025ujunox"
+const CHAIN_RPC_NODE: string = "https://rpc.uni.juno.deuslabs.fi"
+const CHAIN_INFO: ChainInfo = {
+  rpc: "https://rpc.uni.juno.deuslabs.fi",
+  rest: "https://lcd.uni.juno.deuslabs.fi",
+  chainId: "uni-3",
+  chainName: "Juno Testnet",
+  stakeCurrency: {
+    coinDenom: "junox",
+    coinMinimalDenom: "ujunox",
+    coinDecimals: 6,
+  },
+  bip44: {
+    coinType: 118,
+  },
+  bech32Config: Bech32Address.defaultBech32Config("juno"),
+  currencies: [
+    {
+      coinDenom: "junox",
+      coinMinimalDenom: "ujunox",
+      coinDecimals: 6,
+    },
+  ],
+  feeCurrencies: [
+    {
+      coinDenom: "junox",
+      coinMinimalDenom: "ujunox",
+      coinDecimals: 6,
+    },
+  ],
+  coinType: 118,
+  features: ["ibc-transfer", "ibc-go"],
+  // If this field is not provided, Keplr extension will set the default gas price as (low: 0.01, average: 0.025, high: 0.04).
+  gasPriceStep: {
+    low: 0.03,
+    average: 0.04,
+    high: 0.05,
+  },
+}
+
+const AvailableWallets: Wallet[] = [
+  {
+    id: "mobile-web",
+    name: "",
+    description: "",
+    imageUrl: "",
+    isWalletConnect: false,
+    // Will not be shown in the picker since `isMobileWeb` is true.
+    isMobileWeb: true,
```

Comment:
maybe call `shouldForceConnection`?

Reply:
instead, let's add ` type` or `kind` enum property here. for now we will support `keplr` & `walletconnect` enum